tasties_app/views.py

```diff
@@ -1,22 +1,41 @@
 from django.shortcuts import render
 from tasties_app.models import Recipe, Rating
 from django.db.models import Avg
-from collections import OrderedDict
 
 
 def index(request):
     return render(request, 'tasties_app/index.html',)
 
 
 def base(request):
-    return render(request, 'tasties_app/base.html',)
+
+    search_input = request.POST.get('search')
+    if search_input:
+        recipe_object = Recipe.objects.filter(title=search_input)
```

Comment:
What you're doing here is a naive filtering of Recipes that have titles that match `search` exactly. This is probably not what we want, consider the situation where we would search for "soup" for example - and find absolutely nothing, despite having recipes with titles containig the regex "soup" (in some form). [this](https://stackabuse.com/python-check-if-string-contains-substring/#:~:text=substring))%0A%23%201-,Regular%20Expressions%20(RegEx),-Regular%20expressions%20provide) could be what we need instead of the naive `Recipe.objects.filter(title=search_input)`; You might have to iterate through the entire `Recipe.objects.all()`, though not necessarily. Please look into this.